### ISSUE TITLE
Triangle thumb node for slider

### ISSUE THREAD

**Issue by marlitas** [phetsims/mean-share-and-balance#24]
While researching a bug on the 3D water cups slider I discovered that Blackbody spectrum also has a triangular thumb node for their slider: 
<img width="350" alt="Screen Shot 2022-06-02 at 3 28 14 PM" src="https://user-images.githubusercontent.com/42476338/171741661-183f6a9d-1022-4627-9283-ded4aeef7aa1.png">
Our triangular thumb node looks like this:
<img width="255" alt="Screen Shot 2022-06-02 at 3 31 24 PM" src="https://user-images.githubusercontent.com/42476338/171741888-f31b36c2-aa85-4a95-be25-94fbfefd4670.png">

Should a triangular thumb node for sliders be implemented into scenery-phet to avoid code duplication?

**Comment by samreid** [comment]
Yes, it seems at least some code would ideally be shared between those sims.
